src/App.js

```diff
@@ -1,11 +1,57 @@
 import "./App.css";
+import Header from "./components/Header.js";
+import ToDoList from "./components/ToDoList.js";
+import { useState } from "react";
+
+// const toDoArray = [
+//   { name: "shoppingItem", status: "Pending" },
+//   { name: "running", status: "Done" },
+// ];
+
+export default function App() {
+  const [toDo, setToDo] = useState([]);
+
+  function handleAddToDo(toDoItem) {
+    const newToDo = [
+      ...toDo,
+      {
+        name: toDoItem,
+        status: false,
+      },
+    ];
+    setToDo(newToDo);
+  }
+
+  function handleDeleteToDo(name) {
+    const removedToDo = toDo.filter((toDo) => toDo.name !== name);
```

Comment:
When you call `filter` on an array, what you get back is another array. In this case, you are getting all the todos but the one with the name you got in the function.
For that reason, i'd call this array just `newTodos` or `filteredTodos`, and not `removedTodo`